repose-aggregator/core/repose-core-api/src/main/resources/META-INF/schema/container/container-configuration.xsd

```diff
@@ -214,7 +213,30 @@
                     </xs:documentation>
                 </xs:annotation>
             </xs:element>
-
+            <xs:element name="truststore-filename" type="xs:anyURI" minOccurs="0" maxOccurs="1">
+                <xs:annotation>
+                    <xs:documentation>
+                        <html:p>
+                            The application truststore used for validating client certificates.
+                            This is typically set to the same path as the keystore.
+                            NOTE: This element is only used if the need-client-auth attribute is True.
+                            NOTE: IF the need-client-auth attribute is True AND this element is not present,
+                                  THEN the keystore will be used.
```

Comment:
Should add an assertion that says if the attribute is true at least one of the two is specified